SwiftLeeds/Views/Sponsors/SponsorsView.swift

```diff
@@ -26,10 +28,13 @@ struct SponsorsView: View {
                 case .platinum:
                     Section(header: sectionHeader(for: section.type)) {
                         ForEach(section.sponsors) { sponsor in
-                            contentTile(for: sponsor)
-                                .listRowBackground(Color.clear)
-                                .listRowInsets(EdgeInsets())
-                                .padding(.bottom, 8)
+                            VStack {
+                                contentTile(for: sponsor)
+                                jobtile(for: sponsor)
+                            }
```

Comment:
concern: It's really difficult to see the association between a job and the corresponding sponsor. An easy solution may be to simply add a clear separator between sponsors, but I think ideally the job buttons would be stacked within the sponsor tile itself (keeping a single background per sponsor).